.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,54 @@
+name: "CodeQL"
```

Comment:
`name: CodeQL` syntax should be sufficient or is there some YAML camel case special case?

Reply:
As this was auto-generated by GitHub [during the initialisation](https://docs.github.com/en/github/finding-security-vulnerabilities-and-errors-in-your-code/setting-up-code-scanning-for-a-repository), I would like to keep it this way as it is the suggested way.